demo/Readme.md

```diff
@@ -3,3 +3,15 @@
 ## Warning
 These examples have test case dependencies, think carefuly before changing them
 
+# Generation of grpc client server
+Using Sysl, you can specify endpoints, endpoint behaviour and the underlying data models. The Sysl compiler could automatically generate grpc client and server stubs from a common Sysl specification.
+
+This is how you generate grpc code stubs, it assumes all the Sysl dependencies have been previously installed.
+1. Change into <SYSL_GIT_REPO_ROOT>.
+2. Execute the below command:
+    sysl tmpl --root demo/codegen --root-template demo/codegen --template grpc.sysl --app-name AuthorisationAPI --start start --outdir demo/codegen authorisation
```

Comment:
This command is relative to the root directory (demo/codegen); they should be changed to be executed from the codegen directory

Reply:
its relative to sysl root